keyboards/polykybd/lang/shift_preview.py

```
@@ -0,0 +1,134 @@
+
+_ESCAPES = {'f': 0x0C, 'v': 0x0B, 'b': 0x08, 'n': 0x0A, 'r': 0x0D, 't': 0x09,
+            'a': 0x07, '0': 0x00, '\\': 0x5C, '"': 0x22, "'": 0x27}
+
+
+def _decode_literal(body):
+    """Codepoints of a C `U"..."` body, or None if an escape is not understood."""
+    out, i = [], 0
+    while i < len(body):
+        ch = body[i]
+        if ch != '\\':
+            out.append(ord(ch)); i += 1; continue
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)